LDH/Day4/완주하지 못한 선수.js

```diff
@@ -0,0 +1,22 @@
+function solution(participant, completion) {
+    
+    let win = {};
+    
+    for (let name of participant) {
+        if(win[name]) {
+            win[name] += 1;
+        } else {
+            win[name] = 1;
+        }
+    }
```

Comment:
참가자 수를 셀 때 if...else 문을 사용하여 win 객체에 이름이 있는지 확인하고 카운트를 증가시키고 있습니다. 이 부분을 `||`연산자를 사용하여 개선할 수 있습니다.

- 해당 이름이 win 객체이 있을 때 +1
- win 객체에 없을 때(즉, undefined일 때) 자동으로 0으로 초기화하면서 +1

이렇게 하면 코드가 훨씬 깔끔해집니다.
```suggestion
    for (let name of participant) {
        win[name] = (win[name] || 0) + 1;
    }
```